pennylane_qulacs/qulacs_device.py

```diff
@@ -170,7 +186,7 @@ def apply(self, operations):
 
         for i, op in enumerate(operations):
             # revert the wire numbering such that it adheres to qulacs
-            wires = op.wires.tolist()[::-1]
+            wires = op.wires[::-1]
```

Comment:
I'm not sure about all these following changes. In my thinking, everything the user provides (wires, state-vectors, basis states, unitaries) needs to be reordered, and then qulacs makes computations, and finally we need to reorder the state back to PL convention. Or is this wrong?